drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -54,9 +54,12 @@ class Hmis::Hud::Client < Hmis::Hud::Base
   end
 
   scope :with_access, ->(user, *permissions, **kwargs) do
-    return none unless user.permissions?(*permissions, **kwargs)
+    base_scope = left_outer_joins(:projects).joins(:data_source)
 
-    joins(:data_source).merge(GrdaWarehouse::DataSource.hmis(user))
+    unenrolled_scope = user.permissions?(*permissions, **kwargs) ? base_scope.where(Project: { id: nil }).merge(GrdaWarehouse::DataSource.hmis(user)) : none
+    enrolled_scope = base_scope.merge(Hmis::Hud::Project.with_access(user, *permissions, **kwargs))
+
+    unenrolled_scope.or(enrolled_scope)
   end
```

Comment:
Propose using an `unenrolled` scope that takes WIP-ness into account: https://github.com/greenriver/hmis-warehouse/pull/3004/files#r1180348505

`enrolled_scope = base_scope.merge(..)` also needs to be updated to include projects that the client has WIP enrollments at.